oracles/oracle_spec.go

```diff
@@ -144,6 +144,12 @@ func isInternalOracleData(data OracleData) bool {
 	return true
 }
 
+// matchPubKeys tries to match the public keys from the provided OracleData object with the ones
+// present in the Spec.
+func (s *OracleSpec) matchPubKeys(data OracleData) bool {
```

Comment:
Since it's a method called by an external component, I think we should make this method pubic as it's now part of the API of the oracle spec. It states the intention better.

Reply:
All right, we logically separate them, I understand.
Thanks!

Reply:
This has been corrected here https://github.com/vegaprotocol/vega/pull/5534/commits/587bd850dcca9027fd5422ef1cf618ff087a449d#diff-c31fe3f1776ec18d9a8482d6f1b00732f84023362e6183a17e18e379b0b28ebcR147-R152